doc/changes/DM-33887.bugfix.rst

```diff
@@ -0,0 +1 @@
+HTCondor plugin now correctly passes attributes defined in site's 'profile' section to submit jobs.
```

Comment:
English:  passes..to submit jobs. 
=> passes...to the HTCondor jobs   
(or passes...to the HTCondor job submission files)